src/v/raft/errc.h

```diff
@@ -39,6 +39,7 @@ enum class errc : int16_t {
     replicate_batcher_cache_error,
     group_not_exists,
     replicate_first_stage_exception,
+    invalid_truncation_offset
```

Comment:
what does it mean for an offset to be invalid?

Reply:
out of range